src/lib/env/utils.ts

```diff
@@ -1,7 +1,15 @@
 import { join } from 'path';
 
 export function getOsEnv(key: string): string {
-    return process.env[key] as string;
+    if (typeof process.env[key] === 'undefined') {
+        throw new Error('Environment variable ' + key + ' is not set.');
```

Comment:
`Environment variable ${key} is not set.` - is better

Reply:
There are some security concerns (https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/template_strings) related to template strings, so I prefer to to use concatenated strings always. It uses same amount of chars, so I don't really understand this popularity of this form.

Reply:
If you are always using template strings in project, I can change it.

Reply:
Can you quote or create the screenshot with description of "security concerns"? I didn't found anything about it in `en-US` version of documentation.

Reply:
But basically yes, personally I have cofigured tslint with enabled `prefer-template`

Reply:
This page is not translated to english completely, strange:

https://developer.mozilla.org/he/docs/Web/JavaScript/Reference/template_strings#Security

Reply:
However ENV file it is not user input. I know that the server manager should set it, but external user should not have access to it.
So I don't see the problem using it here.